### PR TITLE
Implement rules for CIS OCP Section 2

### PR DESCRIPTION
**Pull request by rhmdnd**
Now that we have a profile and control files for CIS 1.4.0, we can start
wiring up the existing rules.

This commit ports all the existing rules we were using for the CIS
OpenShift profile into the CIS 1.4.0 version.

### CODE REVIEW

File: controls/cis_ocp_1_4_0/section-2.yml [hunk 1]
```diff
@@ -6,34 +6,42 @@ controls:
...
   - id: '2.6'
     title: Ensure that the --peer-auto-tls argument is not set to true
-    status: pending
-    rules: []
+    status: automated
+    rules:
+      - etcd_peer_auto_tls
     level: level_1
   - id: '2.7'
     title: Ensure that a unique Certificate Authority is used for etcd
```
Comment (rhmdnd) [inline controls/cis_ocp_1_4_0/section-2.yml:47]: Looks like we will need a new rule for this one.

File: controls/cis_ocp_1_4_0/section-2.yml [hunk 2]
```diff
@@ -6,38 +6,47 @@ controls:
...
-    status: pending
-    rules: []
+    status: automated
+    rules:
+      - etcd_unique_ca
```
Comment (jhrozek) [inline controls/cis_ocp_1_4_0/section-2.yml:50]: while we're touching the rules, could we add a CCE to this rule?

**Review by jhrozek — COMMENTED** [review]
I think it's fine to add the missing CCE as a separate PR, so feel free to merge. We should just take this opportunity to clean up the rules.

**Review by jhrozek — APPROVED** [review]
LGTM

**Pull request merged**